.github/dependabot.yml

```diff
@@ -24,7 +23,7 @@ updates:
     schedule:
       interval: "daily"
     labels:
-      - toolchain-dependencies
```

Comment:
`toolchain-dependencies` does not exist as label, so reusing an existing label instead